Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.6-alpine
 
-RUN pip install pipenv==8.2.7
+RUN pip install pipenv==2018.11.26
```

Comment:
Does this need to be pinned?

Reply:
Codacy suggests that it should be and seems to be a pattern in other repos also.